eng/Versions.props

```diff
@@ -54,6 +54,9 @@
     <MicrosoftDotNetApiCompatVersion>5.0.0-beta.20506.7</MicrosoftDotNetApiCompatVersion>
     <MicrosoftDotNetBuildTasksFeedVersion>5.0.0-beta.20506.7</MicrosoftDotNetBuildTasksFeedVersion>
     <MicrosoftDotNetCodeAnalysisVersion>5.0.0-beta.20506.7</MicrosoftDotNetCodeAnalysisVersion>
+    <MicrosoftCodeAnalysisNetAnalyzersVersion>6.0.0-preview1.20513.4</MicrosoftCodeAnalysisNetAnalyzersVersion>
+    <MicrosoftCodeAnalysisCSharpCodeStyleVersion>3.8.0-4.20503.2</MicrosoftCodeAnalysisCSharpCodeStyleVersion>
+    <StyleCopAnalyzersVersion>1.2.0-beta.205</StyleCopAnalyzersVersion>
```

Comment:
nit: I don't think these package live in Arcade (as you placed them into the Arcade dependencies section).

Reply:
Moving to the `Libraries dependencies` section then doesn't see anything else better fit